modules/oadp-auth-azure-sts.adoc

```diff
@@ -0,0 +1,258 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-auth-azure-STS_{context}"]
+= Using OADP with Azure Security Token Service authentication
+
+[role="_abstract"]
+You can use Microsoft Entra Workload ID to access Azure storage for {oadp-short} backup and restore operations. This approach uses the signed Kubernetes service account tokens of the OpenShift cluster. These token are automatically rotated every hour and exchanged with the Azure Active Directory (AD) access tokens, eliminating the need for long-term client secrets.
+
+To use the Azure Security Token Service (STS) configuration, you need the `credentialsMode` field set to `Manual` during cluster installation. This approach uses the Cloud Credential Operator (`ccoctl`) to set up the workload identity infrastructure, including the OpenID Connect (OIDC) provider, issuer configuration and user-assigned managed identities.
+
+
+.Prerequisites
+
+* You have an OpenShift cluster installed on Microsoft Azure with Microsoft Entra Workload ID configured.
```

Comment:
link https://docs.okd.io/latest/installing/installing_azure/ipi/installing-azure-customizations.html#installing-azure-with-short-term-creds_installing-azure-customizations would be great